### PR TITLE
Update log-files.md

### PR DESCRIPTION
**Pull request by kellybos**
Please note that this change is still in the process of being deployed. It is expected to be available worldwide in two to three weeks. It's good to have the documentation ready to go, but please check with kellybos@microsoft.com before final publishing.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/log-files.md [hunk 1]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
 
 -   Desktop logs
 
-When creating a support request with Microsoft Support, the support engineer will require the debug logs. Having the debug logs on hand before creating the support request will allow Microsoft to quickly start troubleshooting the problem. **Media** or **desktop** logs are only required if requested by Microsoft.
+Collect and enable logging
+---------------------------
+
+It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple clicks.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:35]:
```suggestion
It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple of clicks.
```

File: Teams/log-files.md [hunk 2]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+
+Windows
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:37]:
```suggestion
Windows:
```

File: Teams/log-files.md [hunk 3]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+Right-click on the Teams icon in the system tray and choosing Collect support files.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:38]:
```suggestion
Right-click on the Teams icon in the system tray and choose **Collect support files**. 
```

File: Teams/log-files.md [hunk 4]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
-When creating a support request with Microsoft Support, the support engineer will require the debug logs. Having the debug logs on hand before creating the support request will allow Microsoft to quickly start troubleshooting the problem. **Media** or **desktop** logs are only required if requested by Microsoft.
+Collect and enable logging
+---------------------------
+
+It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple clicks.
+
+Windows
+Right-click on the Teams icon in the system tray and choosing Collect support files. 
+
+Mac
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:40]:
```suggestion
Mac:
```

File: Teams/log-files.md [hunk 5]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+Select the Help menu and choose Collect support files.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:41]:
```suggestion
Select the Help menu and choose **Collect support files**.
```

File: Teams/log-files.md [hunk 6]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+
+Debug, Desktop and Media logs will be collected in one folder with the name MSTeams Diagnostics Log <local data and time>. This folder can be compressed and shared when you open a support request with Microsoft Support. This folder will contain folders for Desktop, Meeting (Media) and Debug (web). You can collect the files using the keyboard shortcuts -
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:43]:
```suggestion
Debug, Desktop, and Media logs will be collected in one folder with the name MSTeams Diagnostics Log <local data and time>. This folder can be compressed and shared when you open a support request with Microsoft Support. The folder will contain folders for Desktop, Meeting (Media), and Debug (web). You can collect the files using the keyboard shortcuts -
```

File: Teams/log-files.md [hunk 7]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+
+Windows
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:45]:
```suggestion
Windows:
```

File: Teams/log-files.md [hunk 8]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+Crtl + Alt + Shift + 1
+
+Mac
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:48]:
```suggestion
Mac:
```

File: Teams/log-files.md [hunk 9]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+Option + Command + Shift + 1
+
+Media logging is turned off by default. To enable Media logging, users must turn on the option in the Teams client. Go to Settings > General, select the Enable logging for meeting diagnostics (requires restarting Teams). The Teams client must be restarted for logging to begin.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:51]:
```suggestion
Media logging is turned off by default. To enable Media logging, users must turn on the option in the Teams client. Go to **Settings** > **General**, and select **Enable logging for meeting diagnostics (requires restarting Teams)**. The Teams client must be restarted for logging to begin.
```

File: Teams/log-files.md [hunk 10]
```diff
@@ -29,7 +29,30 @@ There are three types of log files automatically produced by the client, which c
...
+
+> [!NOTE]
+>  If Media logging is enabled there will be additional files included in the Meeting folder which are necessary for investigating audio and video issues. If Media logging isnot enabled there will be a limited number of logs available.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:54]:
```suggestion
> If Media logging is enabled, there will be additional files included in the Meeting folder which are necessary for investigating audio and video issues. If Media logging is not enabled, there will be a limited number of logs available.
```

File: Teams/log-files.md [hunk 11]
```diff
@@ -51,7 +74,7 @@ For a complete list of supported operating systems and browsers, see [Get client
 Debug logs
 ---------------------------
 
-These are the most common logs and are required for all Microsoft support cases. Debug logs are produced by the Windows and Mac desktop clients, as well as by browser-based clients. The logs are text based and are read from the bottom-up. They can be read using any text-based editor, and new logs are created when logging into the client.
+See the	Collect and enable logging section for Windows and Mac instrutions. Debug logs are produced by the Windows and Mac desktop clients, as well as by browser-based clients. The logs are text based and are read from the bottom-up. They can be read using any text-based editor, and new logs are created when logging into the client.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:77]:
```suggestion
See the Collect and enable logging section for Windows and Mac instructions. Debug logs are produced by the Windows and Mac desktop clients, as well as by browser-based clients. The logs are text-based and are read from the bottom-up. They can be read using any text-based editor, and new logs are created when logging into the client.
```

File: Teams/log-files.md [hunk 12]
```diff
@@ -61,48 +84,25 @@ Debug logs show the following data flows:
...
--   Browser: You will be prompted to save the debug log to default save location
+To collect logs for Browser:
+      Keyboard shortcut: Crtl + Alt + Shift + 1
+      The files will be available in %userprofile%\Downloads
 
 Media logs
 ---------------------------
 
-Media logs contain diagnostic data about audio, video, and screen sharing in Teams meetings. They are required for support cases that are linked to call-related issues.
+See the	Collect and enable logging section for Windows and Mac instrutions. Media logs contain diagnostic data about audio, video, and screen sharing in Teams meetings. They are required for support cases that are linked to call-related issues.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:98]:
```suggestion
See the _Collect and enable logging_ section for Windows and Mac instructions. Media logs contain diagnostic data about audio, video, and screen sharing in Teams meetings. They are required for support cases that are linked to call-related issues.
```

File: Teams/log-files.md [hunk 13]
```diff
@@ -61,48 +84,25 @@ Debug logs show the following data flows:
...
 
 Media logging is turned off by default. To log diagnostic data for Teams meetings, users must turn on the option in the Teams client. Go to **Settings** > **General**, select the **Enable logging for meeting diagnostics (requires restarting Teams**) check box, restart Teams, and reproduce the issue. 
 
 The following table outlines the media log locations. When you send the log files to Microsoft support, please verify the timestamp of the log files to ensure the logs cover the time frame when you reproduced the issue.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:102]:
```suggestion
When you send the log files to Microsoft support, please verify the timestamp of the log files to ensure the logs cover the time frame when you reproduced the issue.
```

File: Teams/log-files.md [hunk 14]
```diff
@@ -61,48 +84,25 @@ Debug logs show the following data flows:
...
-|---------|---------|
-|Windows     |%appdata%\Microsoft\Teams\media-stack\\*.blog         |
-|            |%appdata%\Microsoft\Teams\skylib\\*.blog
-|            |%appdata%\Microsoft\Teams\media-stack\\*.etl         |
-|Mac OSX     |~/Library/Application Support/Microsoft/Teams/media-stack/*.blog         |
-|            |~/Library/Application Support/Microsoft/Teams/skylib/*.blog         |
-|Linux       |~/.config/Microsoft/Microsoft Teams/media-stack/*.blog         |
-|            |~/.config/Microsoft/Microsoft Teams/skylib/*.blog         |
+To collect logs for Linux:
+The files will be available ~/.config/Microsoft/Microsoft Teams/media-stack/*.blog and ~/.config/Microsoft/Microsoft Teams/skylib/*.blog.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:105]:
```suggestion
The files will be available in ~/.config/Microsoft/Microsoft Teams/media-stack/*.blog and ~/.config/Microsoft/Microsoft Teams/skylib/*.blog.
```

File: Teams/log-files.md [hunk 15]
```diff
@@ -117,27 +117,13 @@ Here's a list of the log files that are generated and the information they conta
 Desktop logs
 ---------------------
 
-Desktop logs, also known as bootstrapper logs, contain log data that occurs between the desktop client and the browser. Like media logs, these logs are only needed if requested by Microsoft. The logs are text based and can be read using any text-based editor in a top-down format.
+See the	Collect and enable logging section for Windows and Mac instrutions. Desktop logs, also known as bootstrapper logs, contain log data that occurs between the desktop client and the browser. Like media logs, these logs are only needed if requested by Microsoft. The logs are text based and can be read using any text-based editor in a top-down format.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:120]:
```suggestion
See the _Collect and enable logging_ section for Windows and Mac instructions. Desktop logs, also known as bootstrapper logs, contain log data that occurs between the desktop client and the browser. Like media logs, these logs are only needed if requested by Microsoft. The logs are text-based and can be read using any text-based editor in a top-down format.
```

File: Teams/log-files.md [hunk 16]
```diff
@@ -117,27 +117,13 @@ Here's a list of the log files that are generated and the information they conta
...
-
- - Click on the **Microsoft Teams** icon in your system tray, and select **Get Logs**.
-
-|Client |Location |
-|---------|---------|
-|Windows     |%appdata%\Microsoft\Teams\logs.txt         |
-|Mac OSX     |~/Library/Application Support/Microsoft/Teams/logs.txt         |
-|Linux       |~/.config/Microsoft/Microsoft Teams/logs.txt         |
+To collect logs for Linux:
+Click on the **Microsoft Teams** icon in your system tray, and select **Get Logs**.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:123]:
```suggestion
Click on the Microsoft Teams icon in your system tray, and select **Get Logs**.
```

File: Teams/log-files.md [hunk 17]
```diff
@@ -117,27 +117,13 @@ Here's a list of the log files that are generated and the information they conta
...
+The files will be available ~/.config/Microsoft/Microsoft Teams/logs.txt.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:124]:
```suggestion
The files will be available in ~/.config/Microsoft/Microsoft Teams/logs.txt.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@kellybos - please merge my suggested changes if you agree. Thanks.

**Comment by JohanFreelancer9** [comment]
@kellybos - There is one suggested change that you have missed. Could you please commit that as well? Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kellybos** [comment]
Please go ahead and update the article for final publishing. The code change will be live next week. Thanks!

**Comment by kellybos** [comment]
@JohanFreelancer9 @SerdarSoysal @scanum Just wanted to make sure you saw my note about these changes going live this week. Will this be possible?

**Comment by JohanFreelancer9** [comment]
@scanum - could you please follow up on this PR? Thanks.

**Comment by kellybos** [comment]
@scanum This code is now live so it would be great if the documentation updates were posted.

**Comment by garycentric** [comment]
@kellybos 

Here's how your description of the location of the log files appears currently:

image

Your angle brackets must be escaped to appear in documentation. When they aren't, they may be interpreted as disallowed HTML, as they're described in a recent sync PR, https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr/pull/8050.

As a Microsoft-credentialed contributor, you are strongly encouraged to work in the private repos rather than the public. The private repos provide checks for build integrity, checks for spelling and grammar, and preview links to view your changes before they're merged. Changes made in the private repos also have a shorter path to publication. 

You can find out how to get started contributing in the private repos in [Set up your accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=master) in the authoring guide. If you have questions about the process, please post in [Ask an Admin](https://aka.ms/askanadmin/) in Teams, or send mail to marveldocs-pubops@microsoft.com.

**Pull request merged**